protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMailboxProcessor.java

```diff
@@ -241,23 +241,21 @@ private void addFlagsResponses(ImapSession session, SelectedMailbox selected, Im
     protected void addFlagsResponses(ImapSession session, SelectedMailbox selected, ImapProcessor.Responder responder, boolean useUid, MessageRange messageSet, MessageManager mailbox, MailboxSession mailboxSession) throws MailboxException {
 
         final MessageResultIterator it = mailbox.getMessages(messageSet, FetchGroupImpl.MINIMAL,  mailboxSession);
+        final boolean qresyncEnabled = EnableProcessor.getEnabledCapabilities(session).contains(ImapConstants.SUPPORTS_QRESYNC);
```

Comment:
Why do we have final variables here? I thought we were not relying on such freaks...

Reply:
In this class almost all local variables are final.
I don't think they are freaks, they deserve some love :heart: